ament_pep257/ament_pep257/main.py

```diff
@@ -73,8 +98,26 @@ def main(argv=sys.argv[1:]):
     if args.xunit_file:
         start_time = time.time()
 
+    args.ignore = ','.join(args.ignore)
+    args.select = ','.join(args.select)
+    args.add_select = ','.join(args.add_select)
+    args.add_ignore = ','.join(args.add_ignore)
+    if not (args.ignore or args.select) and args.convention == 'ament':
+        args.ignore = ','.join(['D100',
+                                'D101',
+                                'D102',
+                                'D103',
+                                'D104',
+                                'D105',
+                                'D106',
+                                'D107',
+                                'D203',
+                                'D212',
+                                'D404'])
```

Comment:
@Arnatious uh, is `flake8` happy with this?

Reply:
It was, but I ran it through black in 6a645fc and it looks better now